app/views/admin/auctions/needs_attention/_payment_needed.html.erb

```diff
@@ -1,4 +1,4 @@
-<table class="usa-table-borderless">
+<table class="usa-table-borderless" id="table-payment">
```

Comment:
yeah I agree. hard to know. I just know that I generally avoid `ids` in any case in css unless it's for jquery so maybe change to a class instead?